go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/awslabs/tc-redirect-tap
 
-go 1.21
+go 1.23
```

Comment:
Prefer to keep the min version changes in a separate PR and not combine with dependabot PR updates. Any reason why you decided to merge in a single PR?

Reply:
CNI plugins update is requiring Go 1.23 because they bumped the min version.